cmd/lifecycle/analyzer.go

```diff
@@ -108,7 +107,7 @@ func (a *analyzeCmd) Exec() error {
 	factory := lifecycle.NewAnalyzerFactory(
 		a.platform.API(),
 		NewCacheHandler(a.keychain),
-		NewConfigHandler(),
+		lifecycle.NewConfigHandler(&cmd.APIVerifier{}),
```

Comment:
Moved the config handler to the lifecycle package so that it could be unit tested